gtsfm/frontend/verifier/loransac.py

```diff
@@ -0,0 +1,197 @@
+"""
+Wrapper about COLMAP's LORANSAC Essential matrix estimation, using pycolmap's pybind API.
+
+LORANSAC paper:
+ftp://cmp.felk.cvut.cz/pub/cmp/articles/matas/chum-dagm03.pdf
+
+On Linux, a python wheel is available:
+(add URL)
```

Comment:
is it available now?

Reply:
indeed it is. just updated the file and docstring